frontend/src/pages/associations/main.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
```

Comment:
We should either capitalize our component filenames, or switch `App.tsx` to `app.tsx`.
I think the capitalization is much more common, even with functional components.
See also https://github.com/airbnb/javascript/tree/master/react#naming